content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -210,4 +197,523 @@ users:
+kind: ValidatingWebhookConfiguration
+...
+webhooks:
+- name: my-webhook.example.com
+  matchPolicy: Equivalent
+  rules:
+  - operations: ["CREATE","UPDATE","DELETE"]
+    apiGroups: ["apps"]
+    apiVersions: ["v1"]
+    resources: ["deployments"]
+    scope: "Namespaced"
+```
+
+### Matching requests: examples
+
+Here are other examples of rules that could be used to specify which resources should be intercepted.
+
+* Match create requests for all resources (but not subresources) in all API groups and versions:
+
+   ```yaml
+   rules:
+   - operations: ["CREATE"]
+     apiGroups: ["*"]
+     apiVersions: ["*"]
+     resources: ["*"]
+     scope: "*"
+   ```
+
+* Match update requests for all `status` subresources in all API groups and versions:
+
+   ```yaml
+   rules:
+   - operations: ["UPDATE"]
+     apiGroups: ["*"]
+     apiVersions: ["*"]
+     resources: ["*/status"]
+     scope: "*"
+   ```
+
+* Match create, update, and delete requests for `apps/v1` `deployments`,
+and if create/update/delete requests are made for `deployments` in other API groups or versions,
+(like `extensions/v1beta1` or `apps/v1beta1`), call the webhook with the `apps/v1` versions of those objects:
+
+   ```yaml
+   # matchPolicy is available in Kubernetes v1.15+
+   matchPolicy: Equivalent
+   rules:
+   - operations: ["CREATE","UPDATE","DELETE"]
+     apiGroups: ["apps"]
+     apiVersions: ["v1"]
+     resources: ["deployments"]
+     scope: "*"
+   ```
+
+### Contacting the webhook
+
+Once the API server has determined a request should be sent to a webhook,
+it needs to know how to contact the webhook. This is specified in the `clientConfig`
+stanza of the webhook configuration.
+
+Webhooks can either be called via a URL or a service reference,
+and can optionally include a custom CA bundle to use to verify the TLS connection.
+
+#### URL
+
+`url` gives the location of the webhook, in standard URL form
+(`scheme://host:port/path`).
+
+The `host` should not refer to a service running in the cluster; use
+a service reference by specifying the `service` field instead.
+The host might be resolved via external DNS in some apiservers
+(e.g., `kube-apiserver` cannot resolve in-cluster DNS as that would 
+be a layering violation). `host` may also be an IP address.
+
+Please note that using `localhost` or `127.0.0.1` as a `host` is
+risky unless you take great care to run this webhook on all hosts
+which run an apiserver which might need to make calls to this
+webhook. Such installs are likely to be non-portable, i.e., not easy
+to turn up in a new cluster.
+
+The scheme must be "https"; the URL must begin with "https://".
+
+A path is optional, and if present may be any string permissible in
+a URL. You may use the path to pass an arbitrary string to the
+webhook, for example, a cluster identifier.
+
+Attempting to use a user or basic auth e.g. "user:password@" is not
+allowed. Fragments ("#...") and query parameters ("?...") are also not
+allowed.
```

Comment:
When I first read "You may use the path to pass an arbitrary string to the webhook" I assumed we were talking about query params, but this paragraph makes it clear that the idea is to literally use just the URL path part. Maybe move the constraints up to before the comment about how we can pass arbitrary strings to the webhook?

Reply:
I just dropped the discussion of the path and added a random path in the example